tests/v2/validation/upgrade/kubernetes_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build upgrade || validation
```

Comment:
similar question to <area>.<engine> earlier, but why not have upgrade.kubernetes here? I may be missing context on what resolution ToU would want / what value this would add

Reply:
I mean I guess, but what other upgrade connotation would we have?

Reply:
Down the road I see a few use cases, but right now this should be fine as long as we can add resolution later. Does that sound good?

Reply:
yeah for sure, this is an easy change.